pkgs/by-name/mt/mtkclient/package.nix

```diff
@@ -0,0 +1,71 @@
+{
+  lib,
+  fetchFromGitHub,
+  makeWrapper,
+  python3,
+  stdenv,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "mtkclient";
+  version = "2.0.1";
+
+  nativeBuildInputs = [
+    makeWrapper
+  ];
+
+  strictDeps = true;
+
+  python = python3.withPackages (
+    ps: with ps; [
+      capstone
+      colorama
+      flake8
+      fusepy
+      pkgs.keystone
```

Comment:
I'm not a expert, but there is a little bit weird, which you are trying to contains a cpp pkgs in python.

I have a couple of guesses about this illogical operation:
1. You mistook [python package keystone](https://pypi.org/project/keystone/) into pkgs.keystone, for this guess, we could prove the hypothesis via open [requirement](https://github.com/bkerler/mtkclient/blob/main/requirements.txt).
3. ~~You actually want to contains pkgs.keystone in your project~~

Reply:
Yes, you are 100% right. I accidentally left it in there. It was meant to only include `keystone-engine`.